modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -730,18 +730,19 @@ describe('TSS Ecdsa Utils:', async function () {
     });
 
     async function setupSignTxRequestNocks(isTxRequest = true) {
-      let response = { txRequests: [{ ...txRequest, transactions: [{ ...txRequest, unsignedTx: { signableHex: txRequest.unsignedTxs[0].signableHex, serializedTxHex: txRequest.unsignedTxs[0].serializedTxHex } }] }] };
+      const derivationPath = '';
```

Comment:
ideally would love to see an example with actual derivation path test case

Reply:
+1

Reply:
I will add additional test in follow up pr BG-67621